### PR TITLE
<, >, |, $: add page

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by Managor** [comment]
I also wanted to make pages for `&`, `{` and `(` but those symbols were so multipurpose that I couldn't come up with a concise way to describe them.

**Comment by Managor** [comment]
Do note that some of these are only reachable by enclosing the symbol in quotes like `tldr "<"`

File: pages/common/>.md
[the remark this answers is not in the document]
Comment (Managor) [reply pages/common/>.md:1]: Good idea. I much prefer the `&>` as it's much more memorable

**Comment by Managor** [comment]
Not sure how I'd fix that

**Comment by acuteenvy** [comment]
In #12091, `:.md` was renamed to `colon.md`. Renaming the files is the only possible solution, as there is no way to force Windows to allow these characters in filenames.

**Comment by Managor** [comment]
https://github.com/tldr-pages/tldr/issues/13898
As a reminder for the future.

**Comment by Managor** [comment]
Would it be reasonable to have hardcoded exceptions in tldr, to redirect requests with forbidden windows filesystem characters to their complementing characters.
As in if I type `tldr :`, the program searches for `tldr colon` or `tldr ">"` gets converted to `tldr greater-than`
I ask this because I like consistency and being consistent with other symbol based queries like `tldr !` would be nice.

File: pages/common/vertical-line.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Vertical line
```
[the remark this answers is not in the document]
Comment (Managor) [reply pages/common/vertical-line.md]: https://en.wikipedia.org/wiki/Vertical_bar

**Comment by Managor** [comment]
After you've removed those unnecessary parts of the urls, add something to tldr bot that automatically detects when specific URLs contains something extra. gnu.org for example always adds www to its urls when you open the link and manned always adds /man/ when navigating there manually.

File: pages/common/greater-than.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Greater than
+
+> Redirect output to a file.
+> More information: <https://gnu.org/software/bash/manual/bash.html#Redirecting-Output>.
+
+- Redirect `stdout` to a file:
+
+`{{command}} > {{path/to/file}}`
+
+- Append a file:
```
Comment (acuteenvy) [inline pages/common/greater-than.md:10]:
```suggestion
- Append to a file:
```

**Review by acuteenvy — APPROVED** [review]

**Review by spageektti — APPROVED** [review]

**Pull request merged**